src/canisters/frontend/ic-certified-assets/src/http.rs

```diff
@@ -173,6 +182,89 @@ impl HttpResponse {
         }
     }
 
+    #[allow(clippy::too_many_arguments)]
+    pub fn build_ok_from_requested_encodings(
+        asset: &Asset,
+        requested_encodings: &Vec<String>,
+        key: &str,
+        chunk_index: usize,
+        certificate_header: Option<&HeaderField>,
+        callback: &Func,
+        etags: &Vec<Hash>,
```

Comment:
Is it more common to pass this way?
```suggestion
        etags: &[Hash],
```